docs/RelayAPI.md

```diff
@@ -1006,6 +1007,66 @@ The `paymentPath` is an object with the following attributes:
 
 ---
 
+### Earned mediation fees of user
+Returns a list of all earned mediation fees for user in a network.
+#### Request
```

Comment:
Before explaining what the endpoint does can you add a short description of what "mediation fees" are?

In the app we have "imbalance fees", "delegation fees" - now we talk about "mediation fees". I think I know what mediation fee is supposed to be, but an outsider will have difficulties with the domain language.